datamover/pkg/drivers/https/prepare.go

```diff
@@ -139,11 +139,24 @@ func getConnLocation(ctx context.Context, conn *pb.Connector) (*LocationInfo, er
 
 	return nil, errors.New("unsupport type")
 }
-
 func getObjs(ctx context.Context, in *pb.RunJobRequest, defaultSrcLoca *LocationInfo, offset, limit int32) ([]*osdss3.Object, error) {
+
 	switch in.SourceConn.Type {
 	case flowtype.STOR_TYPE_OPENSDS:
 		return getOsdsS3Objs(ctx, in, offset, limit)
+	case flowtype.STOR_TYPE_AWS_S3:
```

Comment:
Are these relevant changes for progress?

Reply:
No but when we are using APIs, migration process gives error at the beginning like "unsupported storage type - aws-s3" .
Thanks

Reply:
Ok. Just for more tracking purpose. IMHO, we should open a new issue for migration and then push these under that, if it doesn't impact progress code.

Reply:
Which kind of migration you used? If it is the migration from one OpenSDS bucket to another OpenSDS bucket, then no such error will return.  Actually, those 'unsupported storage type' code is for migration between non OpenSDS bucket which is not supported now. You can not just add codes like this to support it, it need many other works.

Reply:
Okay I will remove this part. Thanks